crates/frui_widgets/src/flex/stack.rs

```diff
@@ -0,0 +1,260 @@
+use crate::alignment::{self, Alignment, AlignmentDirectional, AlignmentGeometry};
+use crate::{BoxLayoutData, LayoutData, WidgetList};
+use frui::prelude::*;
+use std::cell::Ref;
+
+pub enum StackFit {
+    Loose,
+    Expand,
+    Passthrough,
+}
+
+#[derive(MultiChildWidget)]
+pub struct Stack<T: WidgetList> {
+    pub children: T,
+    pub fit: StackFit,
+    pub alignment: &'static dyn AlignmentGeometry,
+}
```

Comment:
It's a small nitpick, but I'd like to have `alignment` take a generic argument `A: AlignmentGeometry` instead of `&'static dyn AlignmentGeometry`.

I made a PR into your branch, see [https://github.com/xanahopper/frui/pull/1](https://github.com/xanahopper/frui/pull/1) :D

Reply:
Yes, maybe we should unify this (I actually prefer dynamic dispatch but both are fine).
But here I think all `AlignmentGeomentry`'s implement are constant values, using `&'static` references may be better?